android-lib/src/main/java/com/kinvey/android/cache/QueryHelper.java

```diff
@@ -118,6 +122,19 @@ private static void in(RealmQuery query, String field, Object params){
         }
     }
 
+    private static void newIn(RealmQuery query,  String field, Object params){
+        query.beginGroup();
+        if (params.getClass().isArray()) {
+            String[] components = (String[]) params;
```

Comment:
If this is not part of the fix for saving and finding collections in Realm, then let's leave it out of this PR.

Reply:
Removed.